subprojects/common/logger/src/main/kotlin/com.avito.logger/Logger.kt

```diff
@@ -2,9 +2,7 @@ package com.avito.logger
 
 interface Logger {
     fun debug(msg: String)
-    // TODO: implement in clients and remove empty implementation
-    fun info(msg: String) {} // empty implementation is for backward compatibility
```

Comment:
Did you add implementations in clients?

Reply:
I consider it would fail if there were no implementations or you talked about some implementations in Avito repo?

Reply:
Yes, we use this interface in Avito. This is why it has a default implementation.

Reply:
I will create a PR with impls